docs/pages/guides/working-with-mesheryctl.md

```diff
@@ -15,6 +15,18 @@ type: Guides
 - Performance management of Service Meshes and Workloads
 
 <!-- Running `reset` will remove all active container instances, prune pulled images and remove any local volumes created by starting Meshery. -->
+
+## Related Guides
+
+- For an exhaustive list of commands and syntax, refer to the **[`mesheryctl` Command Reference](/docs/guides/mesheryctl-commands)**.
```

Comment:
Remove bold style, please. The `a` tag should suffice alone. The color currently being used isn't distinguishable enough on a white background. Let's make a global CSS update.